src/main.cpp

```diff
@@ -35,15 +35,26 @@ int main(int argc, char* argv[])
     backup_camera->init_graphics(renderer);
     printf("Starting threads\n");
     backup_camera->start_threads();
+    bool setFullscreenNext = true;
```

Comment:
I think it'd be better if the `BackupCamera` object kept track of it's own state.